### DISCUSSION TITLE
Remove references to the Discourse and officially deprecate it?

### DISCUSSION THREAD

**Discussion by asumagic** [speechbrain/speechbrain#1729] [General]
The [Discourse](https://speechbrain.discourse.group/) has been rather inactive for a while and very few contributors, if any, reply to user questions there, leading to most questions being unsolved.  
Considering that GitHub issues and Discussions should entirely supersede the Discourse, my opinion is that:

- The Discourse should be officially deprecated, which AFAIK is sort of the internal stance, but that fact is not made obvious,
- References to the Discourse as a support platform (as implied by the `README`) should be removed,
- The roadmap should be migrated into whatever form on GitHub,
- It should be clearly indicated on the Discourse (e.g. on a pinned thread) that new questions and issues should be posted on GitHub,
- Optionally, after a grace period, the Discourse could be made read-only.

**Comment by Adel-Moumen** [comment]
Hello, 

The core team is aware of this problem and we started working on this by first introducing new templates for issues and of course GitHub discussion. 

As we were not really familiar with GitHub discussion we wanted to see if it was really useful or not. It seems (at least from my point of view) that GitHub discussion is really used by the community. As a result, I think we are ready to drop the Discourse but we will need to wait for approval from the speechbrain core team in a meeting. 

The roadmap could be migrated directly in the [#announcement](https://github.com/speechbrain/speechbrain/discussions/categories/announcements). 

I will do what the necessary to solve this problem as soon as possible.
